frontend/src/citizen-frontend/messages/ThreadView.tsx

```diff
@@ -43,6 +45,10 @@ const MessageContent = styled.div`
   white-space: pre-line;
 `
 
+const ReplyToThreadContainer = styled.span`
+  padding-left: 28px;
+`
+
```

Comment:
Super small thing, but you could remove this unnecessary `<span>` and apply this margin directly to the `InlineButton` component with `styled(InlineButton)`.

Reply:
Nice!